src/index.js

```diff
@@ -58,13 +58,15 @@ export class PostmsgTransport {
     if(!e.data || typeof(e.data) !== 'string') {
       return;
     }
+    let data;
     try {
-      const { key, event } = JSON.parse(e.data);
-      if (key === this._key) {
-        this._handler(event);
-      }
+      data = JSON.parse(e.data);
     } catch (e) {
       return null;
     }
+    const { key, event } = data;
```

Comment:
Yeah, but we already check for `null` in [this line](https://github.com/kadirahq/storybook-channel-postmsg/pull/4/files/a5f5dcf388d2d7271b43cfc83f15a3117198e026#diff-1fdf421c05c1140f6d71444ea2b27638R58). Its guaranteed only a string will reach `JSON.parse`.